index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+	<script src="https://kit.fontawesome.com/27ebd9ab14.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<section class="header">
+	
+		<div class="header-page">
+			<a href="#index.html" class="Mylogo"> My Logo</a>
+			<a class="Hamberger"><i class="fa-solid fa-bars"></i></a>
+		</div>
+	
+		<div class="header-container">
+		  <h1 class="title1">I'm Gardimy <br>Glad to see you!</h1>
+		  <p class="title2">
+			I'm a software developer! I can help you build a product , 
+			feature or website Look through some of my work and experience! If you 
+			like what you see and have a project you need coded,
+			 don't hestiate to contact me.
+		  </p>
+
+		  <div class="social-icon">
+			<h2 class="connect" href="#">LET'S CONNECT</h2>
+			<ul>
+			  <li><a href="https://twitter.com/"><img src="Images/twitter.png" alt="Twitter Icon"></a></li>
+			  <li><a href="https://www.linkedin.com/in/gardimy-charles-40b795212/"><img src="Images/linkedin.png" alt="Linkedin Icon"></a></li>
+			  <li><a href="#"><img src="Images/medium.png" alt="Medium Icon"></a></li>
+			  <li><a href="https://github.com/Gardimy"><img src="Images/github.png" alt="GitHub Icon"></a></li>
+			  <li><a href="#"><img src="Images/Fingers icon.png" alt="Fingers Icon"></a></li>
+			</ul>
+		  </div>
+		  </div>
+	</section>
+
+	<div class="work-container">
```

Comment:
Please use a `section` tag here instead. The `section` tag is used to specify major sections of the components of your application.